okrs_2019.md

```diff
@@ -0,0 +1,229 @@
+# SourceCred 2019 OKRs
+
+This document contains SourceCred's objectives, initiatives, and metrics for
+2019, generated as part of Protocol Labs's annual planning process.
+For an overview of SourceCred itself, see [the overview](overview.md).
+
+The purpose of this document is to provide clarity around what the project's
+goals and priorities are, and accountability to see if we are making effective
+progress towards achieving those goals. However, this is not intended to lock
```

Comment:
I might address the reader more directly here with something along the lines of:

> We hope this enables you to jump into SourceCred with a clear sense of our goals for 2019, and how our granular work on a regular basis translates to the larger vision for the project.

Reply:
Done